order/interface.go

```diff
@@ -49,6 +49,16 @@ const (
 	// self channel balance field. Only orders with this version are allowed
 	// to use the self channel balance field.
 	VersionSelfChanBalance Version = 3
+
+	// VersionSidecarChannel is the order version that added sidecar
+	// channels for bid orders. Only orders with this version are allowed
+	// to set the sidecar ticket field on bid orders. Since sidecar orders
+	// also add the feature of push amounts on the leased channels, this
+	// affects makers as well. Makers that don't want to support leasing out
+	// channels with a push amount (because it might screw up their
+	// accounting or whatever) can opt out by explicitly submitting their
+	// ask orders with a version previous to this one.
+	VersionSidecarChannel Version = 4
```

Comment:
Non-blocking as we can change this before we actually deploy it, but should we consider just combining these versions? Or perhaps on the server-side there's a use for having the versions be distinct?